src/Sylius/Bundle/CoreBundle/Migrations/Version20220203115813.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220203115813 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add messenger transport table for doctrine transports.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        if ($this->isUsingDoctrineTransport() && !$schema->hasTable('messenger_messages')) {
+            $this->addSql('CREATE TABLE messenger_messages (id BIGINT AUTO_INCREMENT NOT NULL, body LONGTEXT NOT NULL, headers LONGTEXT NOT NULL, queue_name VARCHAR(255) NOT NULL, created_at DATETIME NOT NULL, available_at DATETIME NOT NULL, delivered_at DATETIME DEFAULT NULL, INDEX IDX_75EA56E016BA31DB (delivered_at), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        }
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        if ($this->isUsingDoctrineTransport()) {
+            $this->addSql('DROP TABLE messenger_messages');
```

Comment:
I'm sceptical 😄 If I understand correctly if one had a table created before, the `up` method would do nothing... but if one then calls the `bin/console do:mig:mig prev` it would delete the table in the database even though it was not created by this migration :dancer: The thing is, is it something we can live with 😄 cc @Sylius/core-team